### DISCUSSION TITLE
how is the framerate with this ?

### DISCUSSION THREAD

**Discussion by rpaillot** [tesla-android/issue-tracker#13] [General]
hi there

before I try this fun solution, may i ask what framerate i'm going to get in optimal conditions ? 
Cause I often use Tesla mirror to mirror my iphone to the tesla screen while driving, mainly for my kids to look at some movies in long trip (I live dangerously ! ;))
With teslamirror the framerate doesnt go over 12-15 fps...
WIth the tesla android solution do you think I can get like 30-35 fps ? 

Thx

Rom

**Comment by mikegapinski** [comment]
it can go to as high as 45 fps when launched before putting the car into drive, if the app restarts while driving lower quality mode is enforced, it's not as bad as 15fps for sure
